### PR TITLE
[Analytics Integrations] Updates to clarify when to filter out worker_subrequests

### PR DESCRIPTION
**Pull request by crwaters16**

### CODE REVIEW

File: products/fundamentals/src/content/data-products/analytics-integrations/sumo-logic/index.md [hunk 1]
```diff
@@ -166,6 +158,11 @@ The default time interval is set to 24 hours. Note that for correct filter calcu
 
 - worker_subrequest
 
+<Aside type="note">
+ 
+If you have [Cloudflare Workers](https://workers.cloudflare.com/) enabled and only want to see requests from end users, use `requestSource` to filter out all Workers subrequests.
```
Comment (tlozoot) [inline products/fundamentals/src/content/data-products/analytics-integrations/sumo-logic/index.md:163]: Maybe mention that `requestSource='eyeball'` is the way to see only the requests from end-users.

**Review by tlozoot — APPROVED** [review]

File: products/fundamentals/src/content/data-products/analytics-integrations/sumo-logic/index.md (hunk 1 above)
Comment (nayaabahsan) [reply products/fundamentals/src/content/data-products/analytics-integrations/sumo-logic/index.md:163]: Taking over from Claire. To clarify, do you mean instead saying something like 
"If you have Cloudflare Workers enabled and only want to see requests from end users, set the filter requestSource = eyeball to filter out all Workers subrequests"

**Comment by KimJ15** [comment]
@crwaters16 I think this is a PR we can merge. Can you verify?

**Comment by KimJ15** [comment]
This PR was opened before the developer doc site was moved to Cloudflare Pages. There are many conflicting branches. I'm going to close it now, but feel free to open a new issue or ask any questions.

**Pull request closed without merging**